ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1689,3 +1689,31 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+
+FCU_DXZTC_DFSS:
+  guid: "38271d04-f2a1-417c-a3af-b2a2299c7979" 
+  description: "FCU with SS, discharge temp sensors, zone control and sensor, filter change"
+  is_canonical: true
+  implements:
+  - FCU
+  - DXZTC
+  - DFSS
+  opt_uses:
+  - panel_controller_status
+  - filter_alarm
+
+FCU_RHDHS_DXDC_DFSS:
+  guid: "958c49e9-464b-4e17-beda-07fbbf1b9e97"  
+  description: "FCU with temperature & pressure monitor, humidity monitor and water leak & status alarm."
+  is_canonical: true
+  implements:
+  - FCU
+  - RHDHS
+  - DXDC
+  - DFSS
+  opt_uses:
+  - water_leak_detection_alarm
+  - failed_alarm
+  - failed_discharge_fan_alarm
+  - high_air_static_pressure_alarm
```

Comment:
quite right sir, there is VAV attached according to our engineer on site. Thanks will use discharge now

Reply:
im confused, there IS a VAV downstream? if there is a VAV downstream, how does it control its fan speed, and where is its static pressure sensor and setpoint?

or is that a typo, that there is no VAV and these are discharge static pressure sensors? 

please confirm. if it serves a VAV, this whole model needs to be discussed.

Reply:
we'll check this sir as I'm not familiar with the hardwares, i'll ask around who is familiar with the devices here

Reply:
Hi sir, our BMS engineers found out that the AHU will only have 3 points:
- On/Off Control
- On/Off Status
- Fault status

The AHU is owned by the landlord and will only have these 3 points. since the AHU is not fully interfaced to the BMS i guess that would also be the same with the VAV and only control/status/fault would be available as well.